docs/reference/operations-guide/federated-operation.md

```diff
@@ -0,0 +1,730 @@
+    end
+```
+
+#### ApplyViewToElementsFunction
+
+This merge function uses the View from the operation and applies it to all of
+the results, meaning the results are now re-aggregated and re-filtered using the
+Schema, locally in the FederatedStore. This makes the results look like they
+came from one graph, rather than getting back an array of Elements from the
+different subgraphs.  
+
+
+??? Example "Example using the GetAllElements operation"
+    ```json
+    {
+        "class": "uk.gov.gchq.gaffer.federatedstore.operation.FederatedOperation",
+        "operation": {
+            "class": "GetAllElements"
+        },
+        "graphIds": ["GraphA","GraphB"],
+        "mergeFunction": {
+            "class": "uk.gov.gchq.gaffer.federatedstore.util.ApplyViewToElementsFunction"
+        }
+    }
+    
+    ```
+    Result:
+    ```json
+    [
+        {
+            "class": "uk.gov.gchq.gaffer.data.element.Edge",
+            "group": "LivesIn",
+            "source": "1",
+            "destination": "3",
+            "directed": true,
+            "matchedVertex": "SOURCE",
+            "properties": {}
+        },
+        {
+            "class": "uk.gov.gchq.gaffer.data.element.Edge",
+            "group": "MakeModel",
+            "source": "4",
+            "destination": "5",
+            "directed": true,
+            "matchedVertex": "SOURCE",
+            "properties": {}
+        },
+        {
+            "class": "uk.gov.gchq.gaffer.data.element.Edge",
+            "group": "WorksAs",
+            "source": "1",
+            "destination": "3",
+            "directed": true,
+            "matchedVertex": "SOURCE",
+            "properties": {}
+        },
+        {
+            "class": "uk.gov.gchq.gaffer.data.element.Edge",
+            "group": "Drives",
+            "source": "1",
+            "destination": "4",
+            "directed": true,
+            "matchedVertex": "SOURCE",
+            "properties": {}
+        },
+        {
+            "class": "uk.gov.gchq.gaffer.data.element.Entity",
+            "group": "Vehicle",
+            "vertex": "4",
+            "properties": {
+            "type": "Digger"
+            }
+        },
+        {
+            "class": "uk.gov.gchq.gaffer.data.element.Entity",
+            "group": "Manufacturer",
+            "vertex": "5",
+            "properties": {
+            "name": "JCB"
+            }
+        },
+        {
+            "class": "uk.gov.gchq.gaffer.data.element.Entity",
+            "group": "Job",
+            "vertex": "2",
+            "properties": {
+            "type": "Builder"
+            }
+        },
+        {
+            "class": "uk.gov.gchq.gaffer.data.element.Entity",
+            "group": "Person",
+            "vertex": "1",
+            "properties": {
+            "name": "Bob"
+            }
+        },
+        {
+            "class": "uk.gov.gchq.gaffer.data.element.Entity",
+            "group": "Person",
+            "vertex": "1",
```

Comment:
Might be worth describing the schema of these graphs. Specifically, the vertex `1` appears twice in the results, but most schemas would aggregate these results together. Perhaps adding a count property which gets summed would be a good way to demonstrate this. The results could be contrasted to `ConcatenateMergeFunction` which would list them as separate entities.